contracts/libraries/AddressLib.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+pragma abicoder v1;
```

Comment:
why enforce this ?

Reply:
Works cheaper without “viaIR” option, makes no sense with “viaIR” option

Reply:
Thanks for answering, TIL 👍